webservice/requirements.txt

```diff
@@ -51,10 +69,13 @@ python-dateutil==2.6.0
 python-editor==1.0.3
 pytz==2016.10
 requests==2.19.1
+s3transfer==0.1.13
 singledispatch==3.4.0.3
 six==1.10.0
 SQLAlchemy==1.1.5
+SQLAlchemy-Utils==0.33.3
```

Comment:
I think Jesse ended up removing SQLAlchemy, and you have now reintroduced a bunch of them. I suspect it's because you don't have his changes... Also, it seems like this file changed a lot in general. Is that because you did a `pip freeze`? Fine if you did, just trying to understand.

I think you should rebase so that you get Jesse's latest changes, then add the libraries you explicitly need, then run `pip freeze`.

Reply:
Yes, my branch was before Jesse's merge to feature/commons. I suggest we rebase once this PR is approved.

Reply:
I think you should do it to get the PR approved, as the dependencies are part of the PR approval.

Reply:
Resolving this; this was my bad, as SQLAlchemy had been removed from cgp-dashboard, not this repo.